c2corg_ui/views/document.py

```diff
@@ -32,7 +32,9 @@ def __init__(self, request):
             'debug': 'debug' in self.request.params,
             'api_url': self.settings['api_url'],
             'ign_api_key': self.settings['ign_api_key'],
-            'bing_api_key': self.settings['bing_api_key']
+            'bing_api_key': self.settings['bing_api_key'],
+            'image_backend_url': self.settings['image_backend_url'],
+            'image_url': self.settings['image_url']
```

Comment:
These variables are already passed to template in notfound view. 404 page works as expected.